tests/integration/templatetags/test_oscar_thumbnail.py

```diff
@@ -59,6 +60,26 @@ def test_oscar_thumbnail_tag_as_context_value(self):
     def test_oscar_thumbnail_sizes(self):
         self._test_oscar_thumbnail_tag_sizes()
 
+    @patch('oscar.templatetags.image_tags.ThumbnailNode._render')
+    def test_ThumbnailNode_doesnt_raise_if_OSCAR_THUMBNAIL_DEBUG_is_false(self, render_mock):
+        render_mock.side_effect = ValueError()
+        with override_settings(OSCAR_THUMBNAIL_DEBUG=True):
+            with self.assertRaises(ValueError):
+                self.template.render(self.context)
+
+        with override_settings(OSCAR_THUMBNAIL_DEBUG=False):
+            self.assertEqual(self.template.render(self.context), '')
+
+    @patch('oscar.templatetags.image_tags.ThumbnailNode._render')
+    def test_ThumbnailNode_defaults_to_DEBUG_if_OSCAR_THUMBNAIL_DEBUG_is_not_set(self, render_mock):
```

Comment:
I don't think this test is necessary - its essentially just testing the internals of `getattr`.

Reply:
I think it's needed because we're testing that the code falls back to using DEBUG and that feature could be implemented in different ways and could also be broken if that code is updated.

Reply:
OK, suggest we split into two tests as above.